R/kde1d.R

```diff
@@ -12,7 +12,11 @@
 #' @param mult positive bandwidth multiplier; the actual bandwidth used is
 #'   \eqn{bw*mult}.
 #' @param bw bandwidth parameter; has to be a positive number or `NA`; the
-#'   latter calls [`KernSmooth::dpik()`] for automatic selection (default).
+#'   latter uses the direct plug-in methodology, where unknown functionals that
```

Comment:
I would simply write 
> uses the direct plug-in methodology of Sheather and Jones (1991).

and add the reference
> Sheather, S. J. and Jones, M. C. (1991). A reliable data-based bandwidth selection method for kernel density estimation. Journal of the Royal Statistical Society, Series B, 53, 683–690.

Reply:
Wilco, I'll modify when you decide for the other comment